### PR TITLE
Add the logo to the readme 😄 

### PR DESCRIPTION
**Pull request by zackkrida**
Why not?

### CODE REVIEW

**Review by dhruvkb — COMMENTED** [review]
It looks really good! We should add it uniformly to all the repos.

It's not very legible on a dark theme, though.
Screenshot 2021-08-12 at 8 10 18 AM

---

We can do something like [Automattic/gutenberg-desktop](https://github.com/Automattic/gutenberg-desktop/) where it's a full-width banner with a fixed background.
Screenshot 2021-08-12 at 8 12 23 AM

File: README.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
+<p align="center">
+  <img src="https://github.com/WordPress/openverse/raw/main/brand/logo.svg" width="400" style="padding: 40px; margin: 0 auto;" />
+  <br />
+  <br />
+  Openverse is a search engine for openly-licensed media.<br />
+</p>
```
Comment (dhruvkb) [inline README.md:6]:
```suggestion
<img src="brand/banner.svg" width="100%"/>

<p align="center">
  Openverse is a search engine for openly-licensed media.
</p>
```

**Review by dhruvkb — COMMENTED** [review]
After merging #46, let's try it once with this change?

**Review by obulat — APPROVED** [review]
Glad to see our logo everywhere. Thank you for considering all color themes, @dhruvkb !

**Comment by zackkrida** [comment]
Looks great with the yellow-bg banner, awesome improvement @dhruvkb

**Pull request merged**